### DISCUSSION TITLE
Project Status?

### DISCUSSION THREAD

**Discussion by Miserlou** [hansihe/live_data#14] [General]
Hey Hans!

Love your work! I had the same problem as you and was about to start down the same path before finding that you've already done a lot of the work! Excellent!

It looks like the project never got published and documented though, so I was wondering what the status was. Was this just a PoC for yourself, or are you looking for contributors to help get it over the finish line? 

I've got quite a lot of scattered thoughts about this approach and would love to pick your brain about what's feasible and what isn't. I see you're in Norway - any chance you're in Bergen? I'd love to buy you a coffee and rap about Phoenix if you're ever available!

Cheers
Rich

**Comment by hansihe** [comment]
I am holding a talk about this project at ElixirConf EU next week actually!

https://www.elixirconf.eu/

I am definitely looking for contributors for the project. It is already quite functional in many ways, but still needs some work in order to be ready for use by others.

Definitely up for talking! I live in Trondheim, but I do have family in Bergen and visit every now and then. The most convenient format might be a video call.

**Comment by Miserlou** [comment]
Oh, sweet! Glad the project isn't abandoned, I think it's got awesome potential. Do you have a demo repo where you show it in action anywhere, or is there any documentation?

Video call sounds perfect, I'll shoot you an email!

**Comment by hansihe** [comment]
There are some demos in this repo: https://github.com/hansihe/phoenix_data_view_demo

There is not much documentation as of now.

**Comment by hoarf** [comment]
Hey I was wondering if you have plans of continuing this further. I think this has potential to become part of regular phoenix live view package if developed further. The focus of the live view team on browser clients is a bit frustrating to be honest and I think is a way better direction.

**Reply by hansihe** [reply]
Thanks for the kind words!

I will be starting at Fresha in a few days, and they are interested in sponsoring development time for this project. I expect a lot to happen here early next year.
